assets/css/components.css

```diff
@@ -0,0 +1,162 @@
+@import url('../../Components/Avatar/avatar.css');
+@import url('../../Components/Badge/badge.css');
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;900&display=swap');
+
+* {
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+    /* font-size: 62.5%; */
```

Comment:
why is this code commented?

Reply:
I initially tried it out, Zoef. But then every css element looked very small. So I commented it for the time being.